test/fixtures/output/billing/list_invalid_format.golden

```diff
@@ -0,0 +1,15 @@
+Error: accepts 2 arg(s), received 1
```

Comment:
This looks more like a test for a missing argument than an invalid format as its name suggests.

Reply:
my mistake right, updating the test to actually test format error.